apps/admin-page/src/app/pages/edit-discount/edit-discount.component.html

```diff
@@ -0,0 +1,78 @@
+<ng-template #header>
+    <span class="mat-body" *ngIf="!isLoading" data-cy="edit-discount-header"
+        >Discount id: <span data-cy="edit-discount-id">{{ id }}</span></span
+    >
+</ng-template>
+
+<button mat-button (click)="historyBack()" class="back-btn" data-cy="back-btn">
+    <mat-icon data-cy="icon">arrow_back_ios</mat-icon> Back
+</button>
+<webstore-item-card
+    title="Edit discount"
+    [content]="content"
+    [header]="header"
+    [isLoading]="isLoading"
+    data-cy="edit-discount-card"
+    class="edit-discount-content"
+>
```

Comment:
Should be a form. So add a <form [formControl]="editDiscountForm" ....> {{ add everything here and change input fields from [formControll]="..." => formControlName="..." }} </form> for an example, check account page. 

Remember to add error messages as well. For examples look at account page.